source/working_with_kayobe.rst

```diff
@@ -37,7 +37,8 @@ Making a Kayobe Checkout
 A Kayobe checkout is made on the Ansible control host.
 
 A Kayobe development environment can easily be set up using a script called
-``beokay``, for example:
+``beokay``, for example. This command may need the ``KAYOBE_VAULT_PASSWORD``
```

Comment:
The guide should explicitly define when the variable is needed (e.g. "will need" instead of "may need").